new.go

```diff
@@ -40,10 +40,9 @@ func reserveSELinuxLabels(store storage.Store, id string) error {
 			} else {
 				b, err := OpenBuilder(store, c.ID)
 				if err != nil {
-					if err == storage.ErrContainerUnknown {
```

Comment:
This error can be wrapped, so I suggest changing the test to
```
if errors.Cause(err) == storage.ErrContainerUnknown || os.IsNotExist(errors.Cause(err)) {
```
or something similar.  We probably also want to check for these at https://github.com/projectatomic/buildah/blob/master/cmd/buildah/common.go#L48.